### PR TITLE
Dependency Updates

### PR DESCRIPTION
**Pull request by joneubank**
Updated many out of date libraries.

Simplified the mongoDB connection script to accommodate changes from newer mongoose version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,31 +48,31 @@
...
     "swagger-ui-express": "^4.3.0",
     "winston": "^3.6.0"
   },
   "prettier": {
-    "printWidth": 100,
+    "printWidth": 120,
     "trailingComma": "all",
-    "singleQuote": true
+    "singleQuote": true,
+    "arrowParens": "always"
```
Comment (justincorrigible) [inline package.json:76]: should this include the use tabs param?

File: test/functional/mongoUtilsTest.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ describe('Test mongo URI construction', () => {
     expect(uri).to.equal('mongodb://localhost:27017/lectern');
   });
 
-  it('Should construct valid mongo connection URI with user and password', () => {
+  it('Should construct valid mongo connection URI', () => {
```
Comment (justincorrigible) [inline test/functional/mongoUtilsTest.ts:56]: good change 👍

**Review by justincorrigible — APPROVED** [review]

**Comment by justincorrigible** [comment]
jenkins tests seem to not be passing?

File: package.json (hunk 1 above)
Comment (joneubank) [reply package.json:76]: I'll do a separate PR and fix all the prettier stuff. thanks.

**Pull request merged**